.travis.yml

```diff
@@ -17,33 +17,57 @@ script:
   - jabba use ${JDK:=adopt@~1.8-0}
   - java -version
   # copy mssql license agreement for tests
-  - cp container-license-acceptance.txt akka-projection-jdbc/src/test/resources/container-license-acceptance.txt
-  - cp container-license-acceptance.txt akka-projection-slick/src/test/resources/container-license-acceptance.txt
+  - cp container-license-acceptance.txt akka-projection-jdbc/src/it/resources/container-license-acceptance.txt
+  - cp container-license-acceptance.txt akka-projection-slick/src/it/resources/container-license-acceptance.txt
   - sbt -jvm-opts .jvmopts-travis "$AKKA_SNAPSHOT" "$ALPAKKA_KAFKA_SNAPSHOT" "$CMD"
 
 jobs:
   include:
     - stage: check
       env: CMD="verifyCodeStyle"
       name: "Code style check. Run locally with: sbt verifyCodeStyle"
-    - env: CMD="++2.12.11 Test/compile"
+    - env: CMD="++2.12.11 IntegrationTest/compile"
```

Comment:
Does `IntegrationTest/compile` triggers `Test/Compile`? 

Otherwise this should be `CMD="++2.12.11 Test/Compile IntegrationTest/compile"`

We want to pre-compile the tests as well, no?

Reply:
Yes, it will compile `Test` sources too.